ecl/hql/hqlgram.y

```diff
@@ -7976,27 +7976,29 @@ simpleDataSet
                             $$.setExpr(createDataset(no_choosesets, $3.getExpr(), $5.getExpr()));
                             $$.setPosition($1);
                         }
-    | DEDUP '(' startTopLeftRightSeqFilter optDedupFlags ')' endTopLeftRightFilter endSelectorSequence
+    | DEDUP '(' startSortOrder startTopLeftRightSeqFilter optDedupFlags ')' endTopLeftRightFilter endSelectorSequence endSortOrder
```

Comment:
Is it necessarily a sort list?

Would : DEDUP(ds, field1, BEST(field2 > 10, field3 < 10, RIGHT)) 

make sense?